src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/AudioObjectStructuredData.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { string, number } from 'prop-types';
+import serialiseForScript from '#lib/utilities/serialiseForScript';
+
+const AudioObjectStructuredData = ({
+  promoBrandTitle,
+  shortSynopsis,
+  durationISO8601,
+  embedUrl,
+  imageUrl,
+  releaseDateTimeStamp,
+}) => {
+  const metadata = {
+    '@context': 'http://schema.org',
+    '@type': 'AudioObject',
+    name: promoBrandTitle,
+    description: shortSynopsis,
+    duration: durationISO8601,
+    thumbnailUrl: imageUrl,
+    uploadDate: new Date(releaseDateTimeStamp).toISOString(),
+    embedURL: embedUrl,
+  };
+
+  return (
+    <Helmet>
+      <script type="application/ld+json">{serialiseForScript(metadata)}</script>
+    </Helmet>
+  );
+};
+
+AudioObjectStructuredData.propTypes = {
+  promoBrandTitle: string,
+  shortSynopsis: string,
+  durationISO8601: string,
+  embedUrl: string,
+  imageUrl: string,
+  releaseDateTimeStamp: number,
+};
+
+AudioObjectStructuredData.defaultProps = {
+  promoBrandTitle: '',
+  shortSynopsis: '',
+  durationISO8601: '',
+  embedUrl: '',
+  imageUrl: '',
+  releaseDateTimeStamp: null,
+};
+
+export default AudioObjectStructuredData;
```

Comment:
Oops I meant we could just call it `AudioObject` rather than appending StructuredData